cmd/cli/cleanup.go

```diff
@@ -64,27 +66,37 @@ var cleanupCmd = &cobra.Command{
 			Credentials: credentials.NewStaticCredentials(awsKeyID, awsSecretKey, ""),
 		}))
 		clusterService := buildAWSClientFromTypes(awsSession, types, logger)
-		//this could probably leverage channels
-		var currentReport *clusterservice.Report
-		for {
-			newReport, err := clusterService.DeleteResourcesForCluster(clusterId, map[string]string{}, dryRun)
+		if watch {
+			err := wait.PollImmediate(30*time.Second, 20*time.Minute, func() (bool, error) {
```

Comment:
Did we test this against a full RHMI install? Is 20 minutes a safe enough timeout?

Reply:
Very good point. I used 20 minutes because it was the number mentioned in the original JIRA. I'll attempt it against a full cluster.